newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/index.js

```diff
@@ -156,7 +156,7 @@ export default class LinkEvent extends React.Component<EventRendererProps, *> {
                 tabIndex={0}
               >
                 {target || (
-                  <Trans>{`<Enter the name of external events>`}</Trans>
+                  <Trans>{`<Enter the name of external events/scene>`}</Trans>
```

Comment:
Let's revert this, we don't want to encourage including scene events.

Reply:
ok